script/create-sitemaps.js

```diff
@@ -0,0 +1,37 @@
+const metalsmithSitemap = require('metalsmith-sitemap');
+const sitemap = require('sitemap');
+
+function createSitemaps(BUILD_OPTIONS) {
+  const hostname =
+    BUILD_OPTIONS.host === 'localhost'
+      ? 'http://localhost'
+      : BUILD_OPTIONS.host;
+
+  const metalsmithSitemapOptions = {
+    hostname,
+    omitIndex: true,
+  };
+
+  return (files, metalsmith, done) => {
+    if (BUILD_OPTIONS['brand-consolidation-enabled']) {
+      // move dynamic sitemap from sitemap.xml to sitemap-dynamic.xml
+      metalsmithSitemapOptions.output = 'sitemap-dynamic.xml';
+
+      // generate sitemap index that references static va sitemap and dynamic metalsmith sitemap
+      const sitemapIndex = sitemap.buildSitemapIndex({
```

Comment:
opted to use a sitemap generating library so the sitemap index will contain the correct host names.